lib/board.rb

```diff
@@ -0,0 +1,61 @@
+class Board
+  attr_accessor :cells
+
+  def initialize
+    @cells = Array.new(9, " ")
+  end
+
+  def reset!
+    @cells.clear
```

Comment:
You can remove `@cells.clear` since the line below will reset the @cells array to empty strings.